### ISSUE TITLE
CREATE OR REPLACE  TABLE  query throws Unparsable section

### ISSUE THREAD

**Issue by lucheeseng827** [sqlfluff/sqlfluff#3288]
### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.


### What Happened

I was trying to run data lake query from Bigquery and lining the query which looked like this 
```
DROP TABLE IF EXISTS `project-a.dataset-a.table-a` ;
CREATE  OR REPLACE EXTERNAL TABLE `project-a.dataset-a.table-a` ( 
columna STRING,
columnb STRING`
OPTIONS (
  format = 'CSV',
  field_delimiter = '|',
  skip_leading_rows = 1,
  uris = [
      'gs://bucket-a/*'
      ]
);
```

### Expected Behaviour

The queries should be accepting create external table or similar  form.

### Observed Behaviour

When lint, the terminal is throwing 

```
== [sql/test/dwh-script.sql] FAIL                                                                                                                                                                                                                                                                                                                                     
L:   2 | P:   1 |  PRS | Line 2, Position 1: Found unparsable section: 'CREATE OR                                                                                                                                                                                                                                                                                            
                       | REPLACE EXTERNAL TABLE `stor...'
                       
```

### How to reproduce

create the script sql  in folder  and run `sqlfluff lint sql/* --dialect bigquery `

### Dialect

bigquery

### Version

0.11.1

### Configuration

[sqlfluff]
exclude_rules = L032, L044, L057, L039, L052, L001
[sqlfluff:rules]
max_line_length = 150

### Are you willing to work on and submit a PR to address the issue?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

**Comment by tunetheweb** [comment]
Your SQL is incorrect.

Replace the backtick (`) on line 4 with a closing bracket and it parses.

```sql
DROP TABLE IF EXISTS `project-a.dataset-a.table-a` ;
CREATE  OR REPLACE EXTERNAL TABLE `project-a.dataset-a.table-a` ( 
columna STRING,
columnb STRING)
OPTIONS (
  format = 'CSV',
  field_delimiter = '|',
  skip_leading_rows = 1,
  uris = [
      'gs://bucket-a/*'
      ]
);
```
